web/translation/translate.en_US.toml

```diff
@@ -12,7 +12,7 @@
 "protocol" = "Protocol"
 "search" = "Search"
 "filter" = "Filter"
-"loading" = "Loading..."
+"loading" = "Loading"
```

Comment:
Not a good replacement IMHO

Reply:
It was based on UI/UX designer opinion.
In other platforms there should be another opinion and reasons.